tests/test_utilities.py

```diff
@@ -1221,3 +1227,47 @@ def test_resolve_removed_courses(
 
         second_audit_summary = course_exporter.resolve_removed_courses(previous_audit_summary)
         assert second_audit_summary == {}
+
+    @ddt.data(
+        (
+            {
+                'mode': 'enroll',
+                'number': 0,
+            },
+            {
+                'mode': '__audit__',
```

Comment:
It doesn't come into play here, but because the double-underscore pattern has specific semantic meaning, I'd prefer that you avoid using it in testing strings.